stl/inc/xlocale

```diff
@@ -1981,6 +1979,7 @@ protected:
         _Cvt = _Lobj._Getcvt();
     }
 
+    // VERITY

```

Comment:
What's the meaning of this comment?

Reply:
Sorry for that, it was for navigation (ctrl+f), apparently I forgot to delete them